README.md

```diff
@@ -1,98 +1,111 @@
+        ]
+    }
+}
+```
+
+When installing this module, it creates media entity types. One issue is when attempting to install the module on an
+install profile such as lightning or the standard install profiles. Each of these include media entity type
+configurations which conflict with the ones in this module. Extra steps are necessary if existing entity bundles already
+exist. One solution is to temporarily move all the config files in this module into the `config/optional` directory and
+then proceed with installation. This should create any missing media bundles and create the entity browsers that are
+included with this module. The reason for the conflicts is due to dependencies of other modules. Other modules and other
+config files can declare dependency on the configurations of this module. If all configurations of this module were
+placed in the `config/optional` directory, it would cause dependency issues during module/profile installation.
+
+Configuration
+---
+
+1. Add the Media Library button to the WYSIWYG as needed and check the box "Embed media".
+1. If the WYSIWYG is configured to limit allowed HTML Tags, ensure the the below is added to the list of allowed tags:
+`<drupal-media data-entity-type data-entity-uuid data-caption data-align data-* class>`
+   * Although `data-*` would allow all data attributes and you wouldn't need to add any others, the UI form validation will throw an error without the above attributes.
+   * `data-caption` is only required if captions are allowed. This has to be added and `data-*` cant be relied on due to hard coded checking of the embed form.
+   * `data-align` is only required if the media can be aligned. Similar issue as `data-align` above.
+
+
+Troubleshooting
+---
+
+If you are experiencing issues with this module try reverting the configuration files. If you are still experiencing
+issues try posting an issue on the GitHub issues page.
+
+Developer
+---
+
+If you wish to develop on this module you will most likely need to compile some new css. Please use the sass structure
+provided and compile with the sass compiler packaged in this module. To install:
+
+```
+npm install
+grunt watch
+ or
+grunt devmode
+```
+
+Contribution / Collaboration
+---
+
+You are welcome to contribute functionality, bug fixes, or documentation to this module. If you would like to suggest a
+fix or new functionality you may add a new issue to the GitHub issue queue or you may fork this repository and submit a
+pull request. For more help please see [GitHub's article on fork, branch, and pull requests](https://help.github.com/articles/using-pull-requests)
+
+
+Releases
+---
+
+Steps to build a new release:
+- Checkout the latest commit from the `8.x-2.x` branch.
+- Create a new branch for the release.
+- Commit any necessary changes to the release branch.
+- Make a PR to merge your release branch into `master`
+- Give the PR a semver-compliant label, e.g., (`patch`, `minor`, `major`)

```

Comment:
This should be automatic via github actions. If the info.yml changes it will automatically add the `patch` label.
One thing to note about that auto labeler, it will only add labels that already exist in the repo's settings. it wont create new labels.